src/textual/command.py

```diff
@@ -559,6 +561,38 @@ def _become_busy() -> None:
 
         self._busy_timer = self.set_timer(self._BUSY_COUNTDOWN, _become_busy)
 
+    def _stop_no_matches_countdown(self) -> None:
+        if self._no_matches_timer is not None:
+            self._no_matches_timer.stop()
+            self._no_matches_timer = None
+
+    _NO_MATCHES_COUNTDOWN: Final[float] = 0.5
+    """How many seconds to wait before showing 'No matches found'"""
+
+    def _start_no_matches_countdown(self, command_list: CommandList) -> None:
```

Comment:
What's the thinking about having the `CommandList` instance be an argument here? The `CommandList` that is part of the command palette should be queryable.

Reply:
You're right!
Nice catch

Reply:
The argument has been removed and the `CommandList` is queried instead